virttest/utils_test/__init__.py

```diff
@@ -1887,7 +1887,7 @@ def __init__(self, stress_type, params, download_url="", make_cmds="",
         self.remote_host = None
         self.copy_files_to = remote.copy_files_to
         self.params = params
-        self.timeout = 60
+        self.timeout = int(self.params.get('timeout', 60))
```

Comment:
@bssrikanth it would be good to have prefix added to timeout as it will be generic cfg param, timeout can be set by other configs too...e:g:- `cmd_timeout`, `stresscmd_timeout`, something like this